lib/smoke_test/base.rb

```diff
@@ -13,6 +13,8 @@ class Base
     def self.from_env
       raise 'TEST_USER & TEST_USER_PASS env variables must be set to run smoke tests' unless ENV['TEST_USER_PASS'] && ENV['TEST_USER']
 
+      # TODO: because the smoke test currently run against the live url, it doesn't make sense to create such job on staging env
+      # it should be changed to use staging url
```

Comment:
Should we have a ticket for this then? 🤔

Reply:
I am going to remove this comment,  had a brief discussion with Ian, it seems like the smoke test is designed like this